config/samples/flinkoperator_v1alpha1_flinkjobcluster.yaml

```diff
@@ -37,10 +37,5 @@ spec:
     className: org.apache.flink.streaming.examples.wordcount.WordCount
     args: ["--input", "./README.txt"]
     parallelism: 2
-    savepointsDir: /tmp
```

Comment:
Are these related to this change?

Reply:
Not really, just want to keep the sample simple.